src/common/string/slugify.ts

```diff
@@ -14,6 +24,7 @@ export const slugify = (value: string, delimiter = "_") => {
       .toString()
       .toLowerCase()
       .replace(p, (c) => b.charAt(a.indexOf(c))) // Replace special characters
+      .replace(/[а-я]/g, (c) => complex_cyrillic[c]) // Replace some cyrillic characters
```

Comment:
```suggestion
      .replace(/[а-я]/g, (c) => complex_cyrillic[c] || "") // Replace some cyrillic characters
```
Probably not needed but I'd prefer to have this as a precaution.